### ISSUE TITLE
Error installing 0.0.20

### ISSUE THREAD

**Issue by digfish** [dmytrostriletskyi/threads-net#32]
Hi

when trying to upgrade to the release 0.0.20 with `pip install --upgrade threads-net` this error occurs:
```
 error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [17 lines of output]
      Traceback (most recent call last):
        File "/home/sam/.local/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
        File "/home/sam/.local/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/home/sam/.local/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
        File "/tmp/pip-build-env-b3236nwp/overlay/local/lib/python3.10/dist-packages/setuptools/build_meta.py", line 341, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=['wheel'])
        File "/tmp/pip-build-env-b3236nwp/overlay/local/lib/python3.10/dist-packages/setuptools/build_meta.py", line 323, in _get_build_requires
          self.run_setup()
        File "/tmp/pip-build-env-b3236nwp/overlay/local/lib/python3.10/dist-packages/setuptools/build_meta.py", line 487, in run_setup
          super(_BuildMetaLegacyBackend,
        File "/tmp/pip-build-env-b3236nwp/overlay/local/lib/python3.10/dist-packages/setuptools/build_meta.py", line 338, in run_setup
          exec(code, locals())
        File "<string>", line 12, in <module>
      FileNotFoundError: [Errno 2] No such file or directory: '.project-version'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

```
the same error pops out both on Windows 10 (Python 3.10.4 anaconda) and Linux (Python 3.10.6 on ubuntu 22 ).
It was working in the previous releases.

**Comment by RealWorldVesper** [comment]
Same during attempt to upgrade

**Comment by FastFingertips** [comment]
`pip install git+https://github.com/dmytrostriletskyi/threads-net.git/ --force`

**Comment by dmytrostriletskyi** [comment]
Now should be fixed by `v0.0.23` — https://github.com/dmytrostriletskyi/threads-net/releases/tag/v0.0.23

```
$ pip3 install threads-net==0.0.23
Collecting threads-net==0.0.23
  Downloading threads-net-0.0.23.tar.gz (44 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 44.5/44.5 kB 1.1 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: requests==2.31.0 in /Users/dmytrostriletskyi/venvs/threads-net-testing/lib/python3.11/site-packages (from threads-net==0.0.23) (2.31.0)
Requirement already satisfied: charset-normalizer<4,>=2 in /Users/dmytrostriletskyi/venvs/threads-net-testing/lib/python3.11/site-packages (from requests==2.31.0->threads-net==0.0.23) (3.2.0)
Requirement already satisfied: idna<4,>=2.5 in /Users/dmytrostriletskyi/venvs/threads-net-testing/lib/python3.11/site-packages (from requests==2.31.0->threads-net==0.0.23) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in /Users/dmytrostriletskyi/venvs/threads-net-testing/lib/python3.11/site-packages (from requests==2.31.0->threads-net==0.0.23) (2.0.3)
Requirement already satisfied: certifi>=2017.4.17 in /Users/dmytrostriletskyi/venvs/threads-net-testing/lib/python3.11/site-packages (from requests==2.31.0->threads-net==0.0.23) (2023.5.7)
Building wheels for collected packages: threads-net
  Building wheel for threads-net (pyproject.toml) ... done
  Created wheel for threads-net: filename=threads_net-0.0.23-py3-none-any.whl size=20794 sha256=167fd3dc9d081ba390e59a925353dc8386b9901efbddde20d8059032751fc382
  Stored in directory: /Users/dmytrostriletskyi/Library/Caches/pip/wheels/4e/79/85/468083da194e3e0a035c7a9e02c41601218507f9a682809dbb
Successfully built threads-net
Installing collected packages: threads-net
  Attempting uninstall: threads-net
    Found existing installation: threads-net 0.0.13
    Uninstalling threads-net-0.0.13:
      Successfully uninstalled threads-net-0.0.13
Successfully installed threads-net-0.0.23
```
